tests/test_activate.py

```diff
@@ -1062,6 +1062,12 @@ def basic_posix(self, shell):
         shell.assert_env_var('?', '0', True)
         # TODO: assert that reactivate worked correctly
 
+        # regression test for #6840
+        shell.sendline('conda install --blah')
+        shell.assert_env_var('?', '2', use_exact=True)
+        shell.sendline('conda list --blah')
+        shell.assert_env_var('?', '2', use_exact=True)
```

Comment:
I'm not sure if the exit code `2` is really correct. http://tldp.org/LDP/abs/html/exitcodes.html
But it's what argparse does.  From https://github.com/python/cpython/blob/v3.6.4/Lib/argparse.py#L2389 via https://github.com/conda/conda/blob/4.4.9/conda/cli/conda_argparse.py#L147